File: Modul-2/soal-3/PRAK203_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
    float a, b, i, j, x, y;
    printf("Input nilai : ");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);
    printf("Output : %.3f\n", ( a - b ) * i / j - ( x + y ));
    return 0;
}

File: Modul-2/soal-3/PRAK203_2410817210017_OrlandoSugian.py
print("Input nilai : ")
inputan = []
while(len(inputan) < 6) :
    x = input()
    x = x.split()
    inputan += x
a = float(inputan[0])
b = float(inputan[1])
i = float(inputan[2])
j = float(inputan[3])
x = float(inputan[4])
y = float(inputan[5])
hasil = (a - b) * i / j - (x + y)
print("Output : ")
print(f"{hasil:.3f} ")